public/class-monk-public.php

```diff
@@ -254,4 +254,13 @@ public function monk_page_on_front_translations( $page_id ) {
 
 		return $page_id;
 	}
+
+	/**
+	 * Prints the hreflang tags according to the current content.
+	 *
+	 * @return string monk_localization_tags filtered
+	 */
+	public function monk_print_localization_tags() {
+		return apply_filters( 'monk_localization_tags', null, array() );
```

Comment:
All the logic to get language data should be here.
The filter must have the contents of `$monk_localization_data` seen above, so any developer can manipulate the localization data and retrieve a custom tag by language.